lib/kokkos/containers/src/Kokkos_TransformDualView.hpp

```diff
@@ -0,0 +1,997 @@
+/*
```

Comment:
Are you going to upstream this header?

Reply:
Yes, though it needs some work to optimize performance, this was the most basic implementation for proof of concept.